accounts/accounts.go

```diff
@@ -1,3 +1,13 @@
+// (c) 2019-2020, Ava Labs, Inc.
+//
+// This file is a derived work, based on the go-ethereum library whose original
+// notices appear below.
+//
+// It is distributed under a license compatible with the licensing terms of the
+// original code from which it is derived.
+//
+// Much love to the original authors for their work.
+
```

Comment:
Usually license headers in golang are one contiguous comment. I'd recommend replacing this `/n` with `//\n`.

Reply:
Good idea. Going to close this PR and redo in another branch.